infra/helper.py

```diff
@@ -121,6 +121,8 @@ def main():  # pylint: disable=too-many-branches,too-many-return-statements,too-
   _add_engine_args(run_fuzzer_parser)
   _add_sanitizer_args(run_fuzzer_parser)
   _add_environment_args(run_fuzzer_parser)
+  run_fuzzer_parser.add_argument('--corpus-dir',
+                                 help='output location for corpus')
```

Comment:
nit: I think in this case it's not "output location", rather something like "directory containing corpus for the fuzz target"

Reply:
Changed to "directory to store corpus for the fuzz target", since run_fuzzer creates it from scratch.

Reply:
ouch, my bad, it's for the `run_fuzzer`, not `coverage`, you're right